Apps/UnitTests/Android/app/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.18)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+project(UnitTestsJNI)
+
+get_filename_component(UNIT_TESTS_DIR "${CMAKE_CURRENT_LIST_DIR}/../../../../.." ABSOLUTE)
+get_filename_component(REPO_ROOT_DIR "${UNIT_TESTS_DIR}/../.." ABSOLUTE)
+
+set(JSRUNTIMEHOST_TESTS OFF) # Turn off the tests folder for Android Studio path
+add_subdirectory(${REPO_ROOT_DIR} "${CMAKE_CURRENT_BINARY_DIR}/BabylonNative")
+
+npm(install "${UNIT_TESTS_DIR}/../" "UnitTests" "--silent")
```

Comment:
I would recommend using `get_filename_component` to get normalized paths so that we don't see `..` in the path and so that the paths are shorter.

Reply:
Fixed.